scripts/utils/pip/torizon_templates_utils/tasks.py

```diff
@@ -927,4 +927,3 @@ def run_task(self, label: str) -> None:
         if _ret.returncode != 0:
             print(f"> TASK [{label}] exited with error code [{_ret.returncode}] <", color=Color.RED)
             raise RuntimeError(f"Error running task: {label}")
-
```

Comment:
I do not get the meaning of this change

Reply:
This was a bug present already in [TOS6](https://github.com/torizon/vscode-torizon-templates/pull/294/commits/597c01804ee0ae7795858a249cbe2493401eb1b6), it's the same fix but on the xonsh script

Reply:
ok, this I got it. But I'm meaning of this line in particular 930

Reply:
oh ok, you removed the commit.